frontend/src/app/patients/Components/ManagePhoneNumbers.tsx

```diff
@@ -192,7 +202,8 @@ const ManagePhoneNumbers: React.FC<Props> = ({
         <div key={idx}>
           <div className="display-flex">
             <Input
-              className="flex-fill"
+              idString={`phoneInput-${idx}`}
+              className={`flex-fill phoneNumberFormElement`}
```

Comment:
Same here... Do we need the `phoneNumberFormElement` class?

Reply:
please ignore [and see above :) ](https://github.com/CDCgov/prime-simplereport/pull/4065#discussion_r937025620)